# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1328042374 -3600
#      Tue Jan 31 21:39:34 2012 +0100
# Branch gui
# Node ID e8ef57b7bd2ac02e285e3c291605fb5a50f39f88
# Parent  d1c45dd1a038f1265d703d7c54e2e1c2015311a8
Removed .hgsub from gui/ and added entries to the already existing .hgsub.

diff --git a/.hgsub b/.hgsub
--- a/.hgsub
+++ b/.hgsub
@@ -1,1 +1,3 @@
 gnulib = [git]git://git.sv.gnu.org/gnulib
+gui/qterminal = [git]https://code.google.com/p/qterminal/
+gui/qirc = [git]https://code.google.com/p/qirc
diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,3 @@
 f9813bce2c06a6130a68db4478d1b16ddadaf276 gnulib
+86a267ce6ab8c3c9fa40f3e0dbc109a26479384e gui/qirc
+4535670b65621538da028b2e54b2ba30b7839f95 gui/qterminal
diff --git a/gui/.hgsub b/gui/.hgsub
deleted file mode 100644
--- a/gui/.hgsub
+++ /dev/null
@@ -1,2 +0,0 @@
-qterminal = [git]https://code.google.com/p/qterminal/
-qirc = [git]https://code.google.com/p/qirc
